src/jupyter/PopulateDatabase.ipynb

```diff
@@ -31,11 +31,11 @@
    "id": "92716416-e32a-4944-9af2-23b9ac53d576",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #144.    async def doit():
</pre><p>This bit is necessary because it is being run as a script not within the nb which has magic for handling async?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Flowminder/flowkit-ui-backend/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Yes exactly. While running in a nb there's a loop running but on command line there isn't.
I wanted both to work.